docs/data-migrations.md

```diff
@@ -24,10 +24,18 @@ php artisan make:migration data_change_name_in_users_table
 
 When creating data migrations it is important to adhere to the following rules:
 
-1. Do **not** use references to classes and facades that could be removed in the future. This includes Eloquent models. Instead, use the `Illuminate\Support\Facades\DB` facade to interact with the database. You may also access the `Laravel\Pennant\Feature` class to interact with feature flagging.
-2. Ensure that the migration is idempotent. This means that the migration can be run multiple times without causing any issues. Tables and columns should be checked for the existence, data checked if it is in the expected state before it is changed, etc.
-3. Do **not** include a `down` method in the migration. Data migrations are intended to be run once and not rolled back. If you need to roll back a data migration, you should create a new data migration to do so.
-4. Do **not** include schema changes in data migrations. Schema changes should be done in schema migrations. Data migrations should only be used to manipulate data.
+1. Do **NOT** use references to classes and facades that could be removed in the future. This includes Eloquent models. Currently, the only accepted classes to use within migrations are as follows:
+    - `Illuminate\Support\Facades\DB`
+        - Used to Interact with the database.
+    - `Laravel\Pennant\Feature`
+        - To interact with feature flagging.
+2. As mentioned above, ensure all possible SQL errors are captured and handled. (UniqueConstraintExeception for example) Also that any database change is surrounded by a transaction to prevent corruption of the query connection.
```

Comment:
```suggestion
2. As mentioned above, ensure all possible SQL errors are captured and handled, such as `UniqueConstraintExeception`. Also ensure that any database change is surrounded by a transaction to prevent corruption of the query connection.
```